gitlens/gitlens-release-notes-current.md

```diff
@@ -16,6 +16,58 @@ Features marked with `PREVIEW` require a GitKraken Account, with access level ba
 
 ---
 
+
+<a id="v15-1"></a>
+
+## Version 15.1
+
+#### Tuesday, June 5th, 2024
+
+We're excited to announce the latest release of GitLens, bringing you enhanced AI integration, a better Launchpad experience, and bug fixes. This focus on expanding AI capabilities, streamlining user experience, and addressing issues reported by our community.
+
+<img src="/wp-content/uploads/gl-15-1-hero.png" class="img-responsive center img-bordered">
+
+### GitHub Copilot and AI Model Integration
+
+GitLens now supports integration with GitHub Copilot and other VS Code extension-provided AI models for its experimental AI features. You can now specify the AI model you want to use through the new `gitlens.ai.experimental.model` setting. When the model is set to `vscode`, specifying the `gitlens.ai.experimental.vscode.model` setting will target the VS Code extension-provided AI model you want to use.
+
+To simplify the overall configuration of AI models, we have unified all settings under: `gitlens.ai.experimental.model`. This replaces the previous provider-specific settings, making it more straightforward to specify the AI model you want to use.
+
+### Launchpad Enhancements
+
+We've made several improvements to the Launchpad to provide a more intuitive and user-friendly experience. The collapsed state of Launchpad groups is now saved between uses, ensuring a consistent view across sessions. The Draft and Pinned categories in the Launchpad always sort their items by date, making it easier to find recent entries. Additionally, the Launchpad and its status bar indicator now provide clear indications when there is an error loading data, keeping you informed of any issues.
+
```

Comment:
I'd also say we improved loading performance and we are continuing to look for ways to improve that future.